cli/src/cmd/forge/snapshot.rs

```diff
@@ -439,4 +455,18 @@ mod tests {
             }
         );
     }
+
+    #[test]
+    fn can_parse_invariant_snapshot_entry() {
+        let s = "Test:deposit() (runs: 256, calls: 100, reverts:200)";
```

Comment:
is the missing space between reverts: and 200 part of the cli output too? if so should prolly just fix that

Reply:
no, but the regex should catch that, but will update